website/src/components/Changelog/Gateway.tsx

```diff
@@ -42,6 +42,9 @@ export default function Gateway() {
           Switches to user-space DNS resolution, allowing for accurate caching
           based on the TTL in the DNS response.
         </ChangeItem>
+        <ChangeItem pull="10538">
+          Allows passing the FIREZONE_TOKEN as a systemd-credential.
```

Comment:
Should there be more user-facing docs for this? I had to spend some time reading about systemd credentials (TIL) before understanding how to use this.

We could also update the systemd-install script but this seems like a relatively recent addition that may not be supported on all major distros yet?

```suggestion
          Supports loading the FIREZONE_TOKEN from a systemd credential file stored in `$CREDENTIALS_DIRECTORY/FIREZONE_TOKEN`.
```

Reply:
I'll add another section to the docs.

The suggested changelog update doesn't really make sense to me because the path _we_ read the credentials from doesn't matter to the user.

The unit file would e.g. refer to `/etc/secrets/firezone_token` or some other path, only readable by root and then use:

```
LoadCredential=FIREZONE_TOKEN:/etc/secrets/firezone_token.txt
```

The only thing the user needs to know is, what the name of the credential is that the Gateway is looking for.

Reply:
> We could also update the systemd-install script but this seems like a relatively recent addition that may not be supported on all major distros yet?

I'll leave that one as is for now. This change is primarily motivated so we can also offer installation from a `.deb` package. That one cannot just embed the token.